src/constants.py

```diff
@@ -5,3 +5,6 @@
 # Inches per meter, as engine units are inches
 ENGINE_UNITS_PER_METER = 1 / 0.0254
 CHANGELOG_RSS_URL = 'https://forums.playdeadlock.com/forums/changelog.10/index.rss'
+
+APP_ID = '1422450'
+DEPOT_ID = '1422456'
```

Comment:
I think these are better suited in a `versioning/constants.py` file. Also means they can be simply imported

Reply:
I don't see a huge reason to need to split up constants given we have so few right now, but fair enough. Probably better to get ahead of the curve on that one haha

Reply:
The PR includes the use of commands that latest version of pandas has

Reply:
Can't see them in the diff, can you link the code?

Reply:
Nevermind, didn't see last comment on moving it to another PR.